experiments/model_c/api.py

```diff
@@ -0,0 +1,79 @@
+"""
+tts package supports using command line or python to call model for training
+or inferencing. Here we deomnstrate how to use python to call models
+"""
+from typing import Optional, TypeAlias
+import sounddevice as sd
+from scipy.io import wavfile
+import torch
+from TTS.api import TTS
+
+DeviceLikeType: TypeAlias = str | torch.device | int
+
+
+def get_device() -> str:
+    """Check if cuda is available and return corresp. string"""
+    device = "cuda" if torch.cuda.is_available() else "cpu"
+    return device
+
+
+def get_tts_model(
+    identifier: Optional[str] = None, device: DeviceLikeType = "cpu"
```

Comment:
Nit: `identifier: str | None = None`

Reply:
or can I type: `identifier: str = None`?

Reply:
`identifier: str` indicated that `identifier` is an instance of `str`. But `None` is not the type of `str`:
```python
>>> isinstance('str', str)
True
>>> isinstance(None, str)
False
```
`identifier: str = None` will fail in typing check from our internal build build system. Just for your reference and I am ok with your suggestion while we don't really apply for typing check currently. FYI.